app/Http/Controllers/ListingController.php

```diff
@@ -6,6 +6,7 @@
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\Gate;
 use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Config;
```

Comment:
Please confirm, where did you use this?

Reply:
Hi sir @ChronoDevs, Good morning. That is for the pagination located in the config/const. :bow:

Reply:
I think it's working without redeclaration

Reply:
I see. Okay sir, I will remove it :blush: